gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.6-bin.zip
```

Comment:
Regarding checksum validation and regenerated wrapper there is a [known issue](https://docs.gradle.org/current/release-notes#known-issues) in 2,6 that have negative performance impact. It should be already fixed in 2.7-rc-1.